.gitignore

```diff
@@ -48,3 +48,9 @@ out
 
 # VIM temp files
 *.swp
+
+.vscode
+
+# These files are either fetched at build time, or generated from the build
+etld/data/public_suffix_list.h
```

Comment:
nit: I usually try to make these absolute paths with respect to the location of the `.gitignore` file (i.e. `/etld/data/public_suffix.h`) in order to avoid unexpected matches somewhere else in the codebase. Not very likely in this case, so feel free to ignore this suggestion, but a good habit IMHO.